src/solvers/changeCobraSolverParams.m

```diff
@@ -56,9 +56,9 @@
 allowedQPparams = {'minNorm', 'printLevel'};
 allowedMILPparams = {'intTol','relMipGapTol','timeLimit','logFile','printLevel'};
 
-% Only LP is currently included
+% Only LP, QP and MILP are currently included

```

Comment:
From my testing, the `global` variables work on each worker. For instance:
```Matlab
initCobraToolbox
global CBT_LP_SOLVER
parpool(2);
parfor i = 1:2
   CBT_LP_SOLVER
end
```
Note: The `global` variable can also be set after the `parpool` and is propagated to all parallel workers.